draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1711,9 +1710,9 @@ user agent MUST process the cookie as follows:
         processing the cookie.
 
     3.  If the cookie was received from a request which is navigating a
-        top-level browsing context {{HTML}} (e.g. if the request's "reserved
+        top-level traversable {{HTML}} (e.g. if the request's "reserved
         client" is either `null` or an environment whose "target browsing
-        context" is a top-level browing context), skip the remaining substeps
+        context" is in a top-level traversable), skip the remaining substeps
```

Comment:
`target browsing context` is defined by FETCH and it doesn't look like it's been changed (yet?)

Reply:
This is https://github.com/whatwg/html/issues/8463 . In the meantime you can use "target browsing context's navigable is a top-level traversable".